### PR TITLE
Fixes gridventory bugs i guess

### PR DESCRIPTION
**Pull request by ChungusGamer666**
Can't stack bullets inside storage
Disassembling stuff inside storage will just drop the components on user's loc

fixes https://github.com/Mojave-Sun/mojave-sun-13/issues/2202
fixes https://github.com/Mojave-Sun/mojave-sun-13/issues/2409

### CODE REVIEW

File: mojave/items/misc/radios.dm [hunk 1]
```diff
@@ -64,6 +64,8 @@
 	..()
 	weapon.play_tool_sound(src)
 	if(do_after(user, 30 SECONDS, target = src, interaction_key = DOAFTER_SOURCE_DECON))
+		var/drop_location = user.drop_location()
```
Comment (Hekzder) [inline mojave/items/misc/radios.dm]: This is for ham radios, not handhelds. You can't disassemble handhelds. This isn't really needed and will be kind of wonky.

File: mojave/items/misc/fluff.dm [hunk 2]
```diff
@@ -90,10 +90,11 @@
 /obj/item/ms13/fluff/microscope/screwdriver_act_secondary(mob/living/user, obj/item/weapon)
 	user.show_message(span_notice("You begin disassembling \the [src] into scrap."), MSG_VISUAL)
 	if(do_after(user, 8 SECONDS, target = src, interaction_key = DOAFTER_SOURCE_DECON))
+		var/drop_location = user.drop_location()
```
Comment (Hekzder) [inline mojave/items/misc/fluff.dm:94]: Why not define this on the fluff base type?

File: mojave/items/misc/fluff.dm
Comment (Hekzder) [inline mojave/items/misc/fluff.dm:1]: Probably an oversight but it seems like you missed the type writer. It can be picked up and disassembled as well.

**Review by Hekzder — CHANGES_REQUESTED** [review]

File: mojave/items/misc/fluff.dm (hunk 2 above)
Comment (ChungusGamer666) [reply mojave/items/misc/fluff.dm:93]: good question

File: mojave/items/misc/fluff.dm
Comment (ChungusGamer666) [reply mojave/items/misc/fluff.dm:1]: will fix those too

File: mojave/items/misc/radios.dm (hunk 1 above)
Comment (ChungusGamer666) [reply mojave/items/misc/radios.dm:67]: so true

File: mojave/items/misc/fluff.dm (hunk 2 above)
Comment (ChungusGamer666) [reply mojave/items/misc/fluff.dm:93]: not my problem tho imma be real

**Review by Hekzder — APPROVED** [review]
cool swag

**Pull request merged**